lang/java/reef-applications/reef-vortex/pom.xml

```diff
@@ -51,48 +51,20 @@ under the License.
             <version>${project.version}</version>
         </dependency>
         <dependency>
-            <groupId>org.apache.avro</groupId>
-            <artifactId>avro</artifactId>
+            <groupId>com.esotericsoftware</groupId>
+            <artifactId>kryo-shaded</artifactId>
```

Comment:
Good catch! This is BSD-licensed, so I believe we can use it as we're currently using protobuf, which is also another BSD-licensed code. `kryo-serializers` is Apache-licensed, so it should be good to use as well.

I checked how Spark handles this and it seems to be keeping non-Apache licenses of its dependencies in a directory named `licenses`(e.g., protobuf license text in `LICENSE-protobuf.txt`) and maintaining pointers to them in their own `LICENSE` file. Shouldn't we do something similar? Or is there something that I'm missing?